sriov/tests/SR_IOV_macAddress_IPv6/README.md

```diff
@@ -0,0 +1,34 @@
+
+## Test Case Name: SR-IOV.macAddress.IPv6
+
+### Objective(s): Test and ensure that VF (virtual function) admin MAC address functions as intended with IPv6.
+
+### Test procedure
+
+* On trafficgen, clean up existing IPv6 address and neighbor discovery(ND) cache, and add an IPv6 address,
+```
+ip -6 add del ${TGEN_IPv6}/64 dev ${TGEN_PF}
+ip -6 neigh del ${DUT_IPv6} dev ${TGEN_PF}
+ip -6 add add ${TGEN_IPv6}/64 dev ${TGEN_PF}
```

Comment:
I would spell commands completely (ip address), at least in the README file.

Reply:
ip address? we have already used a variable name to represent it for all test cases, in out google doc as well.

Reply:
I see. Sure thing.